apps/api/src/bff/api/endpoints/carts/handlers/create-cart.handler.ts

```
@@ -0,0 +1,24 @@
+import { IControllerHandler } from 'src/shared/controller-handler.interface';
+import { CartFacade } from 'src/modules/cart/application/cart.facade';
+import { CreateCartDto } from '../dto/create-cart.dto';
+
+@Injectable()
+export class CreateCartHandler implements IControllerHandler {
+  constructor(private readonly cartFacade: CartFacade) {}
+
+  async handle(
+    userId: string,
+    dto: CreateCartDto,
+  ): Promise<{ id: string }> {
```

ESLint

[error] Replace `⏎····userId:·string,⏎····dto:·CreateCartDto,⏎··` with `userId:·string,·dto:·CreateCartDto`

(prettier/prettier)